### ISSUE TITLE
set Qwant as the default for Tor Windows (German & France)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#1514]
## Description 

Qwant needs to be used as the default search engine for `Tor Window` for users using France/Germany as their region. We currently set Qwant as the default for everything but `Tor Window`.

## Steps to Reproduce

Assumes you're using Windows and have the following prerequisites:

* change the `Country or region` via the `Region & Language settings` to `France`
* ensure that `France` is set as the `Home Location` via `Region` under `Control Panel`
* Download/Install the `Francais (France)` language via `Language` under `Region & Language settings`
* Restart the machine/VM

Once the above has been completed, you can follow:

   1. ensure that you're launching brave from a brand new profile
   2. once launched, you should notice that Qwant is being used for regular tabs/windows and Incognito 
   3. open a `Tor Window` using `Profile menu` -> `Open Tor Window`

You'll notice that DDG is being used and not Qwant.

## Actual result:

![qwanttor](https://user-images.githubusercontent.com/2602313/46779060-5bd38a00-cce4-11e8-89b2-48231ab648b1.gif)

## Expected result:

France/German regions should be using Qwant globally, including `Tor Window` which is using DDG.

## Reproduces how often: 

100% reproducible using the above STR.

## Brave version (chrome://version info)

Brave | 0.55.12 Chromium: 70.0.3538.45 (Official Build) beta(64-bit)
-- | --
Revision | cbdc32e4334458954e9def214d7e5fa1ca1960eb-refs/branch-heads/3538@{#830}
OS | Mac OS X

### Reproducible on current release:

- Does it reproduce on brave-browser dev/beta builds? 

Yes, reproduced using both `dev` and `beta` builds:

Brave | 0.56.2 Chromium: 70.0.3538.35 (Official Build) dev (64-bit)
-- | --
Revision | 28dcb499844fa40c28d5f62e337876cb936f79f5-refs/branch-heads/3538@{#678}
OS | Mac OS X

Brave | 0.55.12 Chromium: 70.0.3538.45 (Official Build) beta(64-bit)
-- | --
Revision | cbdc32e4334458954e9def214d7e5fa1ca1960eb-refs/branch-heads/3538@{#830}
OS | Mac OS X

- Does it reproduce on browser-laptop?

Nope, works on b-l but the Tor implementations differ between the two.

**Comment by riastradh-brave** [comment]
blocked on https://github.com/brave/brave-browser/issues/1607

**Comment by srirambv** [comment]
Verification Passed on 

Brave | 0.55.17 Chromium: 70.0.3538.67 (Official Build) (64-bit)
-- | --
Revision | 9ab0cfab84ded083718d3a4ff830726efd38869f-refs/branch-heads/3538@{#1002}
OS | Linux

* Verified default search engine for France is Qwant
![image](https://user-images.githubusercontent.com/17010094/47128339-cc3b5780-d25e-11e8-8c29-346529bec335.png)
* Verified default search engine for Germany is Qwant
![image](https://user-images.githubusercontent.com/17010094/47128269-8bdbd980-d25e-11e8-9255-3cf07a0245c7.png)

**Comment by davidrenne** [comment]
I dont fully understand this issue, I just want TOR tabs to be able to set the country of origin.  Is this possible in brave?

**Comment by bsclifton** [comment]
Hi @davidrenne - this issue is tracking that folks in Germany or France will have Qwant as the default search engine when they open a Tor window and then do a search in the omnibox

When you have a Tor window open, you can verify it's working by visiting:
https://check.torproject.org/

I'm not sure of a reliable way to know where the exit is. But you can request a new circuit using Hamburger menu and picking `New Tor connection for this Site`

**Comment by davidrenne** [comment]
No what im trying to ask is, I want the ability for Brave to be able to edit my torrc file

/Users/davidrenne/Library/Application Support/TorBrowser-Data/Tor/torrc

Is the path to edit this file in the Tor browser.

I want something in the GUI of the config or want to know how to edit this file in Brave's system.

I like adding only USA servers in TOR via this config.

`ExitNodes {us}
StrictNodes 1`

I can create a new ticket on this, or maybe you can help me find the path of the torrc file in Brave.  I am new to tor clients as well, but initially it was frustrating being in different countries IPs websites block traffic from random countries your IP ends up going in.  You know?

**Comment by tomlowenthal** [comment]
@davidrenne That's not what this issue is about. We don't currently have plans to implement the feature you describe for a variety of reasons.

**Comment by davidrenne** [comment]
> @davidrenne That's not what this issue is about. We don't currently have plans to implement the feature you describe for a variety of reasons.

Yea I knew it wasnt what this issue was about, I was just trying to get any answers about the tor support for geeky stuff like  ExitNodes etc.

Thanks for the replies.
